pbj-core/pbj-runtime/src/main/java/com/hedera/pbj/runtime/ProtoWriterTools.java

```diff
@@ -805,7 +825,7 @@ private static int sizeOfUnsignedVarInt32(final int value) {
      * @param value The int value to get encoded size for
      * @return the number of bytes for encoded value
      */
-    private static int sizeOfUnsignedVarInt64(long value) {
+    public static int sizeOfUnsignedVarInt64(long value) {
```

Comment:
Do these two methods need to be public, given that `sizeOfVarIn32/64()` is now available?

Reply:
You're right, they don't need to be. Fixed